spec/hydra/pcdm/models/versioned_file_spec.rb

```diff
@@ -0,0 +1,52 @@
+require 'spec_helper'
+
+describe Hydra::PCDM::File do
+
+  let(:file) { Hydra::PCDM::VersionedFile.new }
+  let(:reloaded) { Hydra::PCDM::VersionedFile.new(file.uri) }
+
+  it "has_many_versions" do
+    expect(file.class).to be_versionable
+  end
+
+  describe "when saving" do
+    it "sets the PCDMTerms.File RDF type" do
+      file.content = 'stuff'
+      expect(file.save).to be true
```

Comment:
I'm wondering what this test adds given you already have a test on the following line that matches the description given in the string passed to `it`. I might be inclined to move the two file operations here (adding content, saving) into a `before` block within the "when saving" `describe` block.